Sources/Sentry/SentryDsn.m

```diff
@@ -74,7 +74,7 @@ - (NSURL *)getBaseEndpoint
     components.host = url.host;
     components.port = url.port;
     components.path = [NSString stringWithFormat:@"%@/api/%@/", path, projectId];
-    return components.URL;
+    return SENTRY_UNWRAP_NULLABLE(NSURL, components.URL);
```

Comment:
To be honest, this is unsafe, but I think it's overkill to change this right now. Has been this way for a while